client/src/components/BarChart.js

```diff
@@ -50,7 +51,10 @@ const BarChart = ({
 
     const xScale = d3.scaleBand().domain(
       data.map(function(d) {
-        xLabels[getPropValue(d, xProp)] = getPropValue(d, label)
+        xLabels[getPropValue(d, xProp)] = getPropValue(d, label).substring(
+          0,
+          10
+        ) // TODO: Make responsive
```

Comment:
Well, I don't see that as something related to this pull request.

But I can do that, I would than want to make an ellipsizeString(value, maxLength) util function, and use that also for the OrganizationalChart and for the Kanban, where we also do ellipsizing.
But @VassilIordanov, if we have 10 characters available, do we want to be 10 including the ellipsis or excluding it? And, if it will be 13 including the ellipsis, would that still fit?

Reply:
For the ellipsis itself, please use the proper UTF-8 character: … (\u2026 http://www.fileformat.info/info/unicode/char/2026/index.htm ).

Reply:
Re: _"I don't see that as something related to this pull request."_, this PR truncates the label that was shown fully before.

Reply:
Ideally, I think we want something responsive that truncates the text based on the available space and not on a fixed size, and then as suggested to add a mouseover/tooltip. But I suggest to do that in a separate PR. In this PR we can have indeed an ellipsis util with `&hellip;`

Reply:
Fixed it in this commit: https://github.com/NCI-Agency/anet/pull/3069/commits/954d25bbfc14204719974b7bd8aebdf90b394a4e